l1-contracts/contracts/bridge/L1NativeTokenVault.sol

```diff
@@ -151,25 +151,29 @@ contract L1NativeTokenVault is
     }
 
     /* solhint-disable no-unused-vars */
-    function bridgeMint(uint256 _chainId, bytes32 _assetInfo, bytes calldata _data) external payable override {
-        // if (!hyperbridgingEnabled[_chainId]) { // ToDo: add logic & uncomment
-        //     // Add back
-        //     // Check that the chain has sufficient balance
-        //     require(chainBalance[_chainId][l1Token] >= _amount, "NTV not enough funds 2"); // not enough funds
-        //     chainBalance[_chainId][l1Token] -= _amount;
-        // }
+    function bridgeMint(
+        uint256 _chainId,
+        bytes32 _assetInfo,
+        bytes calldata _data
+    ) external payable override returns (address l1Receiver) {
         address l1Token = tokenAddress[_assetInfo];
-        (uint256 _amount, address l1Receiver) = abi.decode(_data, (uint256, address));
+        (uint256 _amount, address _l1Receiver) = abi.decode(_data, (uint256, address));
```

Comment:
just in case, the variables within functions typically do not use `_` as prefix (we only use it for private functions / function params)

Reply:
I treated it basically as function input, because we decode the bytes to get it. Lets resolve in the base PR, as there are many more cases there.

Reply:
updated anyways